products/cloudfunctions/api.yaml

```diff
@@ -136,17 +136,30 @@ objects:
         description: |
           The Google Cloud Storage URL, starting with gs://, pointing to the zip
           archive which contains the function.
+        exactly_one_of:
```

Comment:
These changes actually don't do anything, because this resource is not currently generated.  I removed the update from the upgrade guide, but I figured I could leave them all in here so that when we eventually do generate it, they'll be there.